### PR TITLE
add local file to ios shoppe app

### PR DESCRIPTION
**Pull request by HaroldFS**
- add local file to ios shoppe app so it can be used without a network call
- moved products from being a global variable to being a local var in the closure
- update closure to pass an array of products
- add a new functions to grab to parse data from local js

### CODE REVIEW

File: swift/ios-shoppe-demo/Resources/ShoppeItems.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+[
```
Comment (patrick-fs) [inline swift/ios-shoppe-demo/Resources/ShoppeItems.js]: I've added an ID parameter to my mock data set: https://github.com/fullstorydev/react-shoppe-demo/blob/master/public/data/products.json

You can just replace this file with that. Also, this is a `.json` file, not a `.js` file.

File: swift/ios-shoppe-demo/ServiceManagers/APIService.swift [hunk 2]
```diff
@@ -33,14 +33,32 @@ class APIService {
...
-                completion()
+                completion(nil)
             }
         }.resume()
     }
+
+    func getProductsFromFile(completion: (_ :[Product])->()) {
+        var products = [Product]()
+
+        if let path = Bundle.main.path(forResource: "ShoppeItems", ofType: "js") {
```
Comment (patrick-fs) [inline swift/ios-shoppe-demo/ServiceManagers/APIService.swift]: once the products file is updated to `.json` extensions, I imagine the `ofType` property will change.

**Review by patrick-fs — APPROVED** [review]
LGTM

File: swift/ios-shoppe-demo/Resources/ShoppeItems.js (hunk 1 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/Resources/ShoppeItems.js]: Perfect I'll knock that out!

File: swift/ios-shoppe-demo/ServiceManagers/APIService.swift (hunk 2 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/ServiceManagers/APIService.swift]: 100%

**Pull request merged**